src/content/contributing/release-process/_index.en.md

```diff
@@ -4,154 +4,209 @@ date: 2020-03-18T16:03:26+01:00
+The typical workflow is to first create release candidate(s) for testing before creating the final release. The suggested
+naming convention is to append `-rcN` to the final version, for example `v0.5.0-rc0`, `v0.5.0-rc1` etc. 
 
-> https://quay.io/repository/submariner/submariner?tab=tags
+Sometimes you may want to create a specific project release for testing prior to creating a release candidate. In this
+case, the suggested naming convention is to append `-preN`.
 
-<!-- TODO(mangelajo) https://github.com/submariner-io/submariner-website/issues/46 -->
+## Create Submariner product release
 
+The following outlines the steps to be taken in order to create a full Submariner product release. As an example, we'll
+use version `v0.5.0`.
 
-## Step 2: Create a Lighthouse Release
+Project releases can either be created via the git CLI or the GitHub UI. This guide uses the GitHub UI as it is
+simpler:
 
-To create lighthouse release artifacts follow the steps below.
+1) Navigate to the project's main releases page.
 
-### Build Lighthouse Agent and CoreDNS
+2) Click `Draft a new release`
 
-Assuming that you have an existing lighthouse git directory, run the following steps .
+3) Fill in the `Tag version` field with the new version.
 
-```bash
-cd lighthouse
-git stash
-git remote add upstream ssh://git@github.com/submariner-io/lighthouse
-git fetch -a -v -t upstream
-git checkout remotes/upstream/master -B master
-git tag -s -m "Globalnet Overlapping IP support RC0" v0.2.0-rc0
-git push upstream v0.2.0-rc0
-```
+4) Fill in the `Release title` field as appropriate. Typically this is just the version (sans the leading `v`) as a release
+   usually contains various changes. But if it's a targeted release then the `Release title` can reflect that.
+   
+5) Enter information to describe the release. This is optional for pre-releases but should be filled in for
+   a final release.
+   
+6) If this is not a final release, mark the checkbox `This is a pre-release`.
 
-A tagged release should appear [here](https://github.com/submariner-io/lighthouse/tags)
+7) Click `Publish release`.
 
-> https://github.com/submariner-io/lighthouse/tags
+For most projects a GitHub action job will be initiated to build release artifacts and publish to Quay. This will take
+several minutes. You can monitor the progress from the project's main page. A small yellow circle should be present to
+the right of the heading above the file listing which indicates it's in progress. You can click it to see details. When
+complete, it will change to either a green check mark or a red X.
 
-A build for v0.2.0-rc0 should start and appear under the "Active branches" section [here](https://travis-ci.com/github/submariner-io/lighthouse/branches)
+### Step 0: Create a `subctl` pre-release, optional
```

Comment:
Shouldn't this come before ` ## Create Submariner product release ` since otherwise your release jobs wont be using the latest subctl?

Or perhaps clarify in that section that it's the general way of creating a release, not instructions to actually start doing. I can see how someone will read this guide and just start that section on `Submariner` project before reading the rest..

Reply:
> Shouldn't this come before `## Create Submariner product release` since otherwise your release jobs wont be using the latest subctl?

I consider it part of creating a Submariner product release, ie it's an initial step that may or may not be need to be done, as indicated by the conditional semantics like "optional", so I don't think it should be before it.  

> 
> Or perhaps clarify in that section that it's the general way of creating a release, not instructions to actually start doing. 

I assume you mean don't include it as a Step. We could include it as part of the previous section but it may get missed if someone skims the previous sections and goes right to the Steps. I prefer it as a Step. We could just make it Step 1 and/or don't treat it as optional since it doesn't hurt to create the pre-release and would be safest to just do it. 

> I can see how someone will read this guide and just start that section on `Submariner` project before reading the rest..

I'm not sure what you mean here.

Reply:
OK - I get it now - I clarified it in https://github.com/submariner-io/submariner-website/pull/200/commits/bbd59f19485581b5be455acdd0b24d4c236c59da